deploy/LOCAL_INSTALL.md

```diff
@@ -46,7 +73,10 @@ The steps below describe how to annotate a callset and then load it into your on
    ```
    gcloud auth application-default login  
    ```
-1. upload your .vcf.gz callset to a google bucket - for example `gs://your-bucket/data/GRCh37/your-callset.vcf.gz`
+1. upload your .vcf.gz callset to a google bucket - for example `gs://your-bucket/data/GRCh38/your-callset.vcf.gz`
+
+1. if you haven't already, upload the reference data you downloaded above to your own google bucket -
+   for example, `gs://your-bucket/reference_data/GRCh38/all_reference_data/combined_reference_data_grch37.ht`
```

Comment:
Is it correct for the file name `combined_reference_data_grch37.ht` in the example? Maybe `37` should be `38`.

Reply:
fixed